packages/documentation/src/stories/foundation/icons/icons.stories.mdx

```diff
@@ -10,21 +9,248 @@ import './icons.styles.scss';
         hidden: true,
       },
     },
+    viewMode: 'docs',
   }}
 />
 
 # Icons
 
-<div class="form-floating">
-  <input id="icon-filter" type="text" class="form-control" placeholder="Filter icons" />
-  <label for="icon-filter">Filter icons</label>
-</div>
+<p className="lead">This is our self-made icon library, with 800+ icons so far.</p>
 
-<div class="icons my-big">
-  {iconsReport.icons.map(icon => (
-    <div class="icon">
-      <post-icon name={icon.name.replace('.svg', '')} />
-      <span>{icon.name.replace('.svg', '')}</span>
+The icons are designed to work with the Swiss Post Design System components. They are SVGs, so they scale quickly and easily and can be styled with CSS. While they are built for us, they will work in any project.<br/>
+Use them together or without the packages `@swisspost/design-system-styles` and `@swisspost/design-system-components`.
```

Comment:
I don't think we're ready to document usage without the component. Let's focus on that first and extend the documentation when needed.

```suggestion
The icons are designed to work with the Swiss Post Design System components. They are SVGs, so they scale and can be styled with CSS.
```